packages/bruno-app/src/components/Environments/EnvironmentSettings/CreateEnvironment/index.js

```diff
@@ -17,8 +17,8 @@ const CreateEnvironment = ({ collection, onClose }) => {
     },
     validationSchema: Yup.object({
       name: Yup.string()
-        .min(1, 'must be at least 1 character')
-        .max(50, 'must be 50 characters or less')
+        .min(1, 'must be atleast 1 characters')
```

Comment:
what was the intention of this change? The previous wording was grammatically correct.